### DISCUSSION TITLE
Im trapped in a loop !

### DISCUSSION THREAD

**Discussion by blackulaphoto** [AntlerVC/firetable#1383] [Support / Q&A]
so i go rowy sign in go to my project which has all been completely setup mind you , anyway it nakes me choose a theme again whatever ,then says i need to set up cloud functions , click it says cloud has already been set up .. yeah i know , then i go to click the template again and it sends me back to setup cloud options again , and im stuck in this loop where i cant acrually work on my project what gives !!

**Comment by harinij** [comment]
One thing you can do here is to go to project setup page and click check Rowy Run and refresh.

Closing older issue, in case the issue still persists you can reopen by commenting with further detailed explanation including reproduction steps of the issue.
